Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/base/WebAppBasePropertyViewPresenter.java

```diff
@@ -108,8 +106,8 @@ protected WebAppProperty generateProperty(@NotNull final WebAppBase webAppBase,
         return new WebAppProperty(propertyMap);
     }
 
-    protected abstract WebAppBase getWebApp(@NotNull String sid, @NotNull String webAppId,
-                                            @Nullable String name) throws Exception;
+    protected abstract WebAppBase getWebAppBase(@NotNull String sid, @NotNull String webAppId,
+                                                @Nullable String name) throws Exception;
```

Comment:
remove spaces

Reply:
Those spaces are used to align the parameters.